### DISCUSSION TITLE
Some times I get an error when opening a pdf in the browser

### DISCUSSION THREAD

**Discussion by Aster89** [qutebrowser/qutebrowser#6335] [Q&A] [answered]
I have `content.pdfjs` set to true and generally, it works well.

Sometimes, however, I get some errors like this:
![qberror](https://user-images.githubusercontent.com/20521900/112714060-54ae8f80-8ed0-11eb-994f-df9f4d5fca90.png)

(In this case I've clicked the download button from Google Drive's preview of a PDF file.)

I'd say that this error is coming up since I `:set content.pdfjs true` some time ago. I don't think I had ever encountered this error before then.

**Comment by rien333** [comment]
Any example link/pdf you can consistently reproduce this on?

**Reply by Aster89** [reply]
The problem arises independently from a specific link/pdf.

**Comment by The-Compiler** [comment, marked as answer]
I've never seen that, though I personally don't use PDF.js much. Could you do `:report` after this happens so I can take a look at the logs?

**Reply by Aster89** [reply]
In this very moment I've searched for _purely functional data structures_ on [duckduckgo](https://duckduckgo.com/?q=purely+functional+data+structures&ia=shopping&iai=r1-0&page=1&sexp=%7B%22cdrexp%22%3A%22b%22%2C%22biaexp%22%3A%22b%22%2C%22msvrtexp%22%3A%22b%22%7D) and I'm trying to open the first PDF result from [www.cs.cmu.edu/~rwh/theses/okasaki.pdf](www.cs.cmu.edu/~rwh/theses/okasaki.pdf), however I get the error similar to the one in the screenshot.

[Here's what I've copied from `:report`.](https://gist.github.com/Aster89/7892fd284a10834e796540a0bf459e40#file-gistfile1-txt)

**Reply by The-Compiler** [reply]
Thanks! Haven't had a chance to take a closer look yet, but I opened an issue: ["No such file or directory" download error when opening PDF via PDF.js · Issue #6366 · qutebrowser/qutebrowser](https://github.com/qutebrowser/qutebrowser/issues/6366)
